training/creating_dataset.py

```diff
@@ -0,0 +1,125 @@
+import os.path
+import traceback
+from copy import deepcopy
+from typing import List
+
+import pandas as pd
+
+from training.analysis_functions import (
+    import_merged_bestiaries,
+    unpack_column,
+    DataFrameType,
+)
+
+
+def create_dataframe(
+    books: List[str] = [
+        "../pathfinder_2e_data/pathfinder-bestiary.db",
+        "../pathfinder_2e_data/pathfinder-bestiary-2.db",
+        "../pathfinder_2e_data/pathfinder-bestiary-3.db",
+    ],
+    characteristics: List[str] = [
+        "system/abilities",
+        "system/attributes/ac",
+        "system/attributes/hp",
+    ],
+) -> DataFrameType:
+    """
+
+    :param books: list of paths of books to load
+    :param characteristics: list of characteristics to load
+    :return: DataFrame with monsters(nps) form chosen books and with chosen characteristics and their origin book
+    """
+    loading_methods = {
+        "system/abilities": move_values_level_up(value_name="mod"),
+        "system/attributes/ac": load_sub_column_as_value("ac"),
+        "system/attributes/hp": load_sub_column_as_value("hp"),
+    }
+    try:
+        check_paths(books)
+    except:
+        traceback.print_exc()
```

Comment:
Teraz to nie jest skuteczne, bo nawet jak check_paths wykryje nieprawidłowości, to i tak idziemy dalej i wywala nam kolejny błąd w innej części tej funkcji. Można by było przerobić check_books na check_book, gdzie będziemy sprawdzać po jednej książce i jeśli się okaże, że coś jest nie tak z konkretną ścieżką, to oprócz wyprintowania informacji wykluczymy ją z przejścia do dalszych części funkcji